acceptance-setup/__tests__/acceptance.js

```diff
@@ -80,6 +80,7 @@ test("Registry should correctly serve rendered and unrendered components", done
       const bodyVersionless = body
         .replace(semverRegex, "6.6.6")
         .replace(/data-hash=\\\".*?\\\"/, "")
+        .replace(/oc-reactRoot-react-app-.*?\"/g, 'oc-reactRoot-react-app-1.1"')
```

Comment:
I think this improves readability?

```suggestion
        .replace(/oc-reactRoot-react-app-.*?\"/g, 'oc-reactRoot-react-app-randomId"')
```